ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -189,8 +189,10 @@ unzip *.zip ;\\
   }
 
   @action
-  addWorkerTag() {
-    this.workerTags.pushObject(new Tag(this.newWorkerKey, this.newWorkerValue));
+  addWorkerTag(e) {
+    this.workerTags.pushObject(
+      new Tag((this.newWorkerKey = e.key), (this.newWorkerValue = e.value)),
```

Comment:
we can just pass the `e.key` and `e.value`, no need to assign these to `newKeyWorker` and `newWorkerValue`

Reply:
@DhariniJeeva ahh, yes.  Removed both tracked values for `newKeyWorker` and `newWorkerValue` since they are no longer needed.